lib/shopify_api/session.rb

```diff
@@ -84,9 +84,9 @@ def extract_current_session
       end
     end
 
-    def initialize(domain:, token:, api_version:, extra: {})
+    def initialize(domain:, token:, api_version: nil, extra: {})
```

Comment:
Not critical, but we could provide the default in the definition, i.e.:

`def initialize(domain:, token:, api_version: ShopifyAPI::Base.api_version, extra: {})`